plugin-script-r/src/main/java/io/kestra/plugin/scripts/r/Script.java

```diff
@@ -39,6 +39,48 @@
                 "  - Rscript -e 'install.packages(\"lubridate\")'"
             }
         ),
+        @Example(
+            full = true,
+            title = """
+            If you want to generate files in your script to make them available for download and use in downstream tasks, you can leverage the `{{outputDir}}` variable. Files stored in that directory will be persisted in Kestra's internal storage. To access this output in downstream tasks, use the syntax `{{outputs.yourTaskId.outputFiles['yourFileName.fileExtension']}}`.
+            """,
+            code = """
+                id: rCars
+                namespace: dev
+
+                tasks:
+                  - id: r
+                    type: io.kestra.plugin.scripts.r.Script
+                    warningOnStdErr: false
+                    docker:
+                    image: ghcr.io/kestra-io/rdata:latest
```

Comment:
```suggestion
                      image: ghcr.io/kestra-io/rdata:latest
```
Wrong YAML indentation